packages/components/inputs/async-creatable-select-input/src/async-creatable-select-input.tsx

```diff
@@ -108,6 +108,14 @@ type TAsyncCreatableSelectInputProps = {
    * [Props from React select was used](https://react-select.com/props)
    */
   'aria-labelledby'?: ReactSelectAsyncCreatableProps['aria-labelledby'];
+  /**
+   * a11y attribute to determine if input is valid/not
+   */
+  'aria-invalid'?: boolean;
+  /**
+   * a11y attribute identifier that points to a field with error message
+   */
+  'aria-errormessage'?: string;
```

Comment:
Please use the same prop descriptions style as in the other aria props.

```suggestion
  /**
   * Indicate if the value entered in the input is invalid.
   * <br>
   * [Props from React select was used](https://react-select.com/props)
   */
  'aria-invalid'?: boolean;
  /**
   * HTML ID of an element containing an error message related to the input.
   * <br>
   * [Props from React select was used](https://react-select.com/props)
   */
  'aria-errormessage'?: string;
```

Same for the other components (except for the link to react-select, which is only needed for "select" components).